frontend/src/pages/Mint/Group.tsx

```diff
@@ -0,0 +1,86 @@
+import Header from "@/components/Header";
+import {
+  IonContent,
+  IonPage,
+  IonFab,
+  IonFabButton,
+  IonIcon,
+  IonFabList,
+  IonGrid,
+  IonCol,
+  IonProgressBar,
+  IonRow,
+  useIonRouter,
+  IonRouterOutlet,
+  IonLabel,
+  IonTabBar,
+  IonTabButton,
+  IonTabs,
+  IonBackButton,
+  IonButtons,
+} from "@ionic/react";
+import { PageLoadingIndicator } from "@/components/PageLoadingIndicator";
+import { Route, RouteComponentProps } from "react-router";
+import Challenge from "./Challenge";
+import Challenges from "./Challenges";
+import Messages from "./Messages";
+import MyGroups from "./Mint";
+import Info from "./Info";
+import useGetGroupById from "@/hooks/Groups/useGetGroupById";
+
+const Mint: React.FC<RouteComponentProps> = ({ match }) => {
+  const { groupId } = match.params as any;
+  const { group, loading } = useGetGroupById(groupId);
+  return (
+    <IonPage>
+      <Header title={group?.name}>
+        <IonButtons slot="start">
+          <IonBackButton defaultHref="/mint"></IonBackButton>
+        </IonButtons>
+      </Header>
+      <IonContent className="ion-padding" color="light">
+        {loading ? (
+          <PageLoadingIndicator />
+        ) : (
+          <IonTabs>
+            <IonRouterOutlet>
+              <Route
+                path={`/mint/:groupId/challenges`}
```

Comment:
Hm not sure if correct, a challenge is not assosciated with a groupId.

Only pools are assosciated with a groupId.

A challenge is only shown on the discover page. You can create a pool by choosing a challenge. 

This route should be mint/groupId/pools